### ISSUE TITLE
Crash if no audio system found

### ISSUE THREAD

**Issue by Godzil** [kitao/pyxel#10]
I know that I'm in a weird environment where my pc actually have an audio card (on the mainboard) but I don't have the drivers (and don't want them as it is a work PC and audio is not something I need here)

So when Pyxel try to find and get the audio card it "crash":

```
python3 pyxel_examples/01_hello_pyxel.py 
Traceback (most recent call last):
  File "pyxel_examples/01_hello_pyxel.py", line 20, in <module>
    App()
  File "pyxel_examples/01_hello_pyxel.py", line 6, in __init__
    pyxel.init(160, 120, caption='Hello Pyxel')
  File "/usr/local/lib/python3.6/dist-packages/pyxel/__init__.py", line 33, in init
    border_width, border_color)
  File "/usr/local/lib/python3.6/dist-packages/pyxel/app.py", line 87, in __init__
    self._audio_player = AudioPlayer()
  File "/usr/local/lib/python3.6/dist-packages/pyxel/audioplayer.py", line 143, in __init__
    callback=self._output_stream_callback)
  File "/usr/local/lib/python3.6/dist-packages/sounddevice.py", line 1373, in __init__
    **_remove_self(locals()))
  File "/usr/local/lib/python3.6/dist-packages/sounddevice.py", line 696, in __init__
    extra_settings, samplerate)
  File "/usr/local/lib/python3.6/dist-packages/sounddevice.py", line 2489, in _get_stream_parameters
    info = query_devices(device)
  File "/usr/local/lib/python3.6/dist-packages/sounddevice.py", line 487, in query_devices
    raise PortAudioError('Error querying device {0}'.format(device))
sounddevice.PortAudioError: Error querying device -1
```

I think it should gracefully  says that audio is not going to work because it can't find a valid sound card, but should not fail because of that.

**Comment by kitao** [comment]
Pyxel uses the sounddevice (https://github.com/spatialaudio/python-sounddevice/tree/0.3.11), the Python wrapper of PortAudio. It seems that there's no way to detect the audio card is active or not in the soudedevice APIs  (am I right?), so I need to change the soundedevice or detect it before using it.

I'll investigate.

**Comment by Godzil** [comment]
I've seen that there is this function though:

http://python-sounddevice.readthedocs.io/en/0.3.11/#sounddevice.query_devices

Not necessarily usefull here, but if you want to allow at some point the user to select which audio device to use, that could be an idea!

**Comment by kitao** [comment]
Thank you for the information!
Though I haven't released it as python package for PyPI not yet, I fixed it in the master branch.

**Comment by kitao** [comment]
This bug is fixed in 0.7.3.
Please let me know if it doesn't work.
